Puppetfile

```diff
@@ -1,15 +1,14 @@
 forge "http://forge.puppetlabs.com"
 
-mod 'epel',                  :git => 'git://github.com/stahnma/puppet-module-epel.git',                         :ref => '1.1.1'
-mod 'stdlib',                :git => 'git://github.com/puppetlabs/puppetlabs-stdlib.git',                       :ref => '4.6.0'
-mod 'inifile',               :git => 'git://github.com/puppetlabs/puppetlabs-inifile.git',                      :ref => '3863d9a9c1e314be73f3517a481f0ad1f2f7a772' # 1.4.3
+mod 'epel',                  :git => 'git://github.com/stahnma/puppet-module-epel.git',                         :ref => '1.3.0'
+mod 'stdlib',                :git => 'git://github.com/puppetlabs/puppetlabs-stdlib.git',                       :ref => '4.24.0'
+mod 'inifile',               :git => 'git://github.com/puppetlabs/puppetlabs-inifile.git',                      :ref => '2.1.1'
 mod 'firewall',              :git => 'git://github.com/puppetlabs/puppetlabs-firewall.git',                     :ref => '1.7.1'
-mod 'java',                  :git => 'git://github.com/danielparks/puppetlabs-java.git',                        :ref => '07d2d9d79f26922ef5db2115b4f036c8c628a019'
-mod 'jenkins',               :git => 'git://github.com/jenkinsci/puppet-jenkins.git',                           :ref => 'd70fd6f6ccc5a8eae7a9f7c1f7d17efd34ad87e7'
-mod 'archive',               :git => 'git://github.com/voxpupuli/puppet-archive.git',                                     :ref => 'v0.4.8'
-mod 'python',                :git => 'git://github.com/stankevich/puppet-python.git',                           :ref => 'e7389f6e270796372b93b178fd5295abefbdc17a' # 1.10.0
-mod 'rvm',                   :git => 'git://github.com/waynr/puppet-rvm.git',                                   :ref => '53c564e56ff8a9b284dfa0bd03e9b9523cc8c3bb'
-# soon-ish, we should pin this to a tag or a SHA, but I expect a little more churn for a while,
-#  so pointing it to `master` for now.
+mod 'java',                  :git => 'git://github.com/danielparks/puppetlabs-java.git',                        :ref => '1.3.0'
+mod 'jenkins',               :git => 'git://github.com/jenkinsci/puppet-jenkins.git',                           :ref => 'v1.7.0'
+mod 'archive',               :git => 'git://github.com/voxpupuli/puppet-archive.git',                           :ref => 'v2.2.0'
+mod 'python',                :git => 'git://github.com/stankevich/puppet-python.git',                           :ref => '1.18.2'
+mod 'maestrodev-rvm',        :git => 'git://github.com/maestrodev/puppet-rvm',                                  :ref => 'v1.13.1'
+mod 'golja-gnupg',           :git => 'git://github.com/dgolja/golja-gnupg.git'                                  :ref => '1.2.3'
```

Comment:
Shouldn't these be `mod 'rvm'` and `mod 'gnupg` respectively?

Reply:
I got both of those names from their forge pages... but maybe? This also does seem to work.

Reply:
Ahh, it looks like both works and that the forge name is required when installing from the forge but from this* page it looks like its maybe idiomatic to just use the module name when installing from vcs? I don't care which way we go (I'd default to just the mod name, rather than the forge name), but I'd like us to have a consistent pattern within this file.

* https://github.com/puppetlabs/r10k/blob/master/doc/puppetfile.mkd

Reply:
Okay I'll update it to remove the forge name. Thanks for the reference!